image/rhel/create-bundle.sh

```diff
@@ -34,6 +34,8 @@ INPUT_ROOT="$1"
 DATA_IMAGE="$2"
 BUILDER_IMAGE="$3"
 OUTPUT_DIR="$4"
+# Install the PG repo natively if true (versus using a container)
+NATIVE_PG_INSTALL="${5:-false}"
```

Comment:
when it will be set to `false`? Do we use this script in other places?

Reply:
It's `false` e.g. when making local builds. It's only set to `true` on OpenShift CI, where we already have RHEL (and there's no docker server)